docs/generated/go.md

```diff
@@ -0,0 +1,201 @@
+---
+bodyclass: docs
+headline: Go Generated Code Guide
+layout: docs
+sidenav: doc-side-quickstart-nav.html
+type: markdown
+---
+
+<p class="lead">This guide gives an overview of code generated with the protoc grpc go plugin.</p>
+
+<div id="toc"></div>
+
+## Compiler Invocation and Results of Code Generation
+This section provides instructions for using the `grpc` plugin to `protoc-gen-go`, when compiling `.proto` files with `protoc`.
+See https://developers.google.com/protocol-buffers/docs/reference/go-generated for detailed instructions on 
+how to use the `protoc` compiler the `protoc-gen-go` plugin.
+
+When `protoc` is invoked with a `--go_out=plugins=grpc:` argument, gRPC stub and service interface code gets generated
+from the `.proto` files specified in the rest of the command. The names of the output files are computed in the same way as though the "plain" Go
```

Comment:
The names of the output files are computed in the same way as when generating the default protocol buffers code with `protoc-gen-go`.

(we don't need to send them to the protobuf Generated Code guide again, we've already told them where it is)